pkg/model/adminrules/model.go

```diff
@@ -63,7 +74,27 @@ type DecisionPerCapability struct {
 // A list of decisions, e.g. [{"capability": "read", "decision": {"deploy": "True"}}, {"capability": "write", "decision": {"deploy": "False"}}]
 type RuleDecisionList []DecisionPerCapability
 
+type AttributeOptimization struct {
+	// Attribute name
+	// +required
+	Attribute taxonomy.Attribute `json:"attribute"`
+	// Optimization directive: minimize or maximize
+	// +required
+	Directive OptimizationDirective `json:"directive"`
+	// Weight, a positive number not exceeding 1.0
+	// Serialized as a string
+	Weight string `json:"weight,omitempty"`
+}
+
+// A list of attribute optimizations
+type OptimizationStrategy struct {
+	Strategy []AttributeOptimization `json:"strategy"`
+	Policy   DecisionPolicy          `json:"policy"`
+}
+
 // Result of query evaluation
 type EvaluationOutputStructure struct {
```

Comment:
How is this different than the EvaluationOutput in pkg/adminconfig?

Reply:
This is an internal structure received from OPA evaluation, not the final output.